synapse/third_party_rules/access_rules.py

```diff
@@ -733,6 +737,10 @@ def _is_power_level_content_allowed(
         # have a special circumstance, but still want to encourage a certain pattern during
         # room creation.
         if on_room_creation:
+            # We specifically don't fail if "invite" or "state_default" are None, as those
+            # values should be replaced with our "default" power level values anyways,
+            # which are compliant
+
             # If invite requirements are <PL50
             if content.get("invite", 50) < 50:
```

Comment:
I realised we should not use a hardcoded value (50) here but the value defined in `default_power_levels` for `invite`.

Same for the `state_default`value

Reply:
Good point, don't want them to get out of sync.